lms/djangoapps/courseware/views/views.py

```diff
@@ -1613,21 +1613,23 @@ def render_xblock(request, usage_key_string, check_if_enrolled=True):
         return render_to_response('courseware/courseware-chromeless.html', context)
 
 
-@require_http_methods(["GET"])
-@ensure_valid_usage_key
-@xframe_options_exempt
-@transaction.non_atomic_requests
-def render_public_video_xblock(request, usage_key_string):
+def _render_public_video_xblock(request, usage_key_string, is_=False):
     """
-    Returns an HttpResponse with HTML content for the Video xBlock with the given usage_key.
-    The returned HTML is a chromeless rendering of the Video xBlock (excluding content of the containing courseware).
+    Look up a given usage key and render the "public" view or the "embed" view
     """
     view = 'public_view'
+    if is_:
+        template = 'public_video_share_embed.html'
+    else:
+        template = 'public_video.html'
```

Comment:
This is a jank naming convention. Should be something clearer like `embed` or `chromeless`.

```suggestion
    if embed:
        template = 'public_video_share_embed.html'
    else:
        template = 'public_video.html'
```

Reply:
Wow, that's one hell of a jank-a** typo. How in the world did they both end up matching?? Maybe I did a find+replace without noticing? either way, thanks for catching